src/ameba/rule/base.cr

```diff
@@ -115,7 +115,7 @@ module Ameba::Rule
 
     # Adds an issue to the *source*
     macro issue_for(*args, **kwargs, &block)
-      source.add_issue(self, {{ args.splat }}, {{ kwargs.double_splat }}) {{ block }}
+      source.add_issue(self, {{ args.splat(", ") }}{{ kwargs.double_splat }}) {{ block }}
```

Comment:
can you elaborate here please? Does this change the behaviour? What's the reason of this change?

Reply:
Without this change this macro produces invalid code if called with only named arguments.